guides/common/modules/proc_configuring-dns-dhcp-and-tftp.adoc

```diff
@@ -84,4 +75,11 @@ ifeval::["{context}" == "{smart-proxy-context}"]
 ----
 endif::[]
 
-For more information about configuring DHCP, DNS, and TFTP services, see {ProvisioningDocURL}Configuring_Network_Services_provisioning[Configuring Network Services] in _{ProvisioningDocTitle}_.
+.Additional resources
+* To view a complete list of configurable options, enter the `{installer-scenario} --help` command.
```

Comment:
I know this was present here before but is this really the right place for it? The output is, as one might expect, extremely broad and only a very small portion is relevant to this procedure.
```
# satellite-installer --scenario satellite --help
Usage:
    satellite-installer [OPTIONS]

Options:

= Generic:
    --[no-]lock-package-versions                                           Let installer lock versions of the installed packages to prevent
                                                                           unexpected breakages during system updates. The choice is remembered
                                                                           and used in next installer runs. (default: nil)
    --tuning INSTALLATION_SIZE                                             Tune for an installation size. Choices: default, medium, large, extra-large, extra-extra-large, development (default: "development")
    --certs-update-server                                                  This option will enforce an update of the HTTPS certificates (default: false)
    --certs-update-server-ca                                               This option will enforce an update of the CA used for HTTPS certificates. (default: false)
    --certs-update-all                                                     This option will enforce an update of all the certificates for given host (default: false)
    --certs-reset                                                          This option will reset any custom certificates and use the self-signed CA instead. Note that any clients will need to be updated with the latest katello-ca-consumer RPM, and any external proxies will need to have the certs updated by generating a new certs tarball. (default: false)
    --certs-skip-check                                                     This option will cause skipping the certificates sanity check. Use with caution (default: false)
    --register-with-insights                                               This option will register the system with Red Hat Insights. (default: false)
    -i, --interactive                                                      Run in interactive mode
    -n, --noop                                                             Run puppet in noop mode? (default: false)
    -s, --skip-checks-i-know-better                                        Skip all system checks (default: false)
    -v, --[no-]verbose                                                     Display log on STDOUT instead of progressbar (default: true)
    -l, --verbose-log-level LEVEL                                          Log level for log based terminal output.
                                                                           The available levels are ERROR, WARN, NOTICE, INFO, DEBUG. See --full-help for definitions. (default: "notice")
    -S, --scenario SCENARIO                                                Use installation scenario
    --list-scenarios                                                       List available installation scenarios
    -h, --help                                                             print help
    --full-help                                                            print complete help
    --[no-]enable-apache-mod-status                                        Enable 'apache_mod_status' puppet module (default: false)
    --[no-]enable-certs                                                    Enable 'certs' puppet module (default: true)
    --[no-]enable-foreman                                                  Enable 'foreman' puppet module (default: true)
    --[no-]enable-foreman-cli                                              Enable 'foreman_cli' puppet module (default: true)
    --[no-]enable-foreman-cli-ansible                                      Enable 'foreman_cli_ansible' puppet module (default: true)
    --[no-]enable-foreman-cli-azure                                        Enable 'foreman_cli_azure' puppet module (default: true)
    --[no-]enable-foreman-cli-google                                       Enable 'foreman_cli_google' puppet module (default: true)
    --[no-]enable-foreman-cli-katello                                      Enable 'foreman_cli_katello' puppet module (default: true)
    --[no-]enable-foreman-cli-kubevirt                                     Enable 'foreman_cli_kubevirt' puppet module (default: false)
    --[no-]enable-foreman-cli-puppet                                       Enable 'foreman_cli_puppet' puppet module (default: false)
    --[no-]enable-foreman-cli-remote-execution                             Enable 'foreman_cli_remote_execution' puppet module (default: true)
    --[no-]enable-foreman-cli-virt-who-configure                           Enable 'foreman_cli_virt_who_configure' puppet module (default: true)
    --[no-]enable-foreman-cli-webhooks                                     Enable 'foreman_cli_webhooks' puppet module (default: true)
    --[no-]enable-foreman-compute-ec2                                      Enable 'foreman_compute_ec2' puppet module (default: true)
    --[no-]enable-foreman-compute-libvirt                                  Enable 'foreman_compute_libvirt' puppet module (default: true)
    --[no-]enable-foreman-compute-openstack                                Enable 'foreman_compute_openstack' puppet module (default: true)
    --[no-]enable-foreman-compute-ovirt                                    Enable 'foreman_compute_ovirt' puppet module (default: true)
    --[no-]enable-foreman-compute-vmware                                   Enable 'foreman_compute_vmware' puppet module (default: true)
    --[no-]enable-foreman-plugin-ansible                                   Enable 'foreman_plugin_ansible' puppet module (default: true)
    --[no-]enable-foreman-plugin-azure                                     Enable 'foreman_plugin_azure' puppet module (default: true)
    --[no-]enable-foreman-plugin-bootdisk                                  Enable 'foreman_plugin_bootdisk' puppet module (default: false)
    --[no-]enable-foreman-plugin-discovery                                 Enable 'foreman_plugin_discovery' puppet module (default: true)
    --[no-]enable-foreman-plugin-google                                    Enable 'foreman_plugin_google' puppet module (default: true)
    --[no-]enable-foreman-plugin-kubevirt                                  Enable 'foreman_plugin_kubevirt' puppet module (default: false)
    --[no-]enable-foreman-plugin-leapp                                     Enable 'foreman_plugin_leapp' puppet module (default: true)
    --[no-]enable-foreman-plugin-openscap                                  Enable 'foreman_plugin_openscap' puppet module (default: true)
    --[no-]enable-foreman-plugin-puppet                                    Enable 'foreman_plugin_puppet' puppet module (default: false)
    --[no-]enable-foreman-plugin-remote-execution                          Enable 'foreman_plugin_remote_execution' puppet module (default: true)
    --[no-]enable-foreman-plugin-remote-execution-cockpit                  Enable 'foreman_plugin_remote_execution_cockpit' puppet module (default: false)
    --[no-]enable-foreman-plugin-rh-cloud                                  Enable 'foreman_plugin_rh_cloud' puppet module (default: true)
    --[no-]enable-foreman-plugin-tasks                                     Enable 'foreman_plugin_tasks' puppet module (default: true)
    --[no-]enable-foreman-plugin-templates                                 Enable 'foreman_plugin_templates' puppet module (default: true)
    --[no-]enable-foreman-plugin-virt-who-configure                        Enable 'foreman_plugin_virt_who_configure' puppet module (default: true)
    --[no-]enable-foreman-plugin-webhooks                                  Enable 'foreman_plugin_webhooks' puppet module (default: true)
    --[no-]enable-foreman-proxy                                            Enable 'foreman_proxy' puppet module (default: true)
    --[no-]enable-foreman-proxy-content                                    Enable 'foreman_proxy_content' puppet module (default: true)
    --[no-]enable-foreman-proxy-plugin-ansible                             Enable 'foreman_proxy_plugin_ansible' puppet module (default: true)
    --[no-]enable-foreman-proxy-plugin-dhcp-infoblox                       Enable 'foreman_proxy_plugin_dhcp_infoblox' puppet module (default: false)
    --[no-]enable-foreman-proxy-plugin-dhcp-remote-isc                     Enable 'foreman_proxy_plugin_dhcp_remote_isc' puppet module (default: false)
    --[no-]enable-foreman-proxy-plugin-discovery                           Enable 'foreman_proxy_plugin_discovery' puppet module (default: true)
    --[no-]enable-foreman-proxy-plugin-dns-infoblox                        Enable 'foreman_proxy_plugin_dns_infoblox' puppet module (default: false)
    --[no-]enable-foreman-proxy-plugin-openscap                            Enable 'foreman_proxy_plugin_openscap' puppet module (default: true)
    --[no-]enable-foreman-proxy-plugin-remote-execution-script             Enable 'foreman_proxy_plugin_remote_execution_script' puppet module (default: true)
    --[no-]enable-foreman-proxy-plugin-shellhooks                          Enable 'foreman_proxy_plugin_shellhooks' puppet module (default: false)
    --[no-]enable-katello                                                  Enable 'katello' puppet module (default: true)
    --[no-]enable-puppet                                                   Enable 'puppet' puppet module (default: false)


= Module apache_mod_status:
    --apache-mod-status-extended-status                                    Determines whether to track extended status information for each request, via the ExtendedStatus directive. (current: "On")
    --apache-mod-status-requires                                           A Variant type that can be:
                                                                           - String with:
                                                                           - '' or 'unmanaged' - Host auth control done elsewhere
                                                                           - 'ip <List of IPs>' - Allowed IPs/ranges
                                                                           - 'host <List of names>' - Allowed names/domains
                                                                           - 'all [granted|denied]'
                                                                           - Array of strings with ip or host as above
                                                                           - Hash with following keys:
                                                                           - 'requires' - Value => Array as above
                                                                           - 'enforce' - Value => String 'Any', 'All' or 'None'
                                                                           This encloses "Require" directives in "<Require(Any|All|None)>" block
                                                                           Optional - If unspecified, "Require" directives follow current flow (current: UNDEF)
    --apache-mod-status-status-path                                        Path assigned to the Location directive which defines the URL to access the server status. (current: "/server-status")


= Module certs:
    --certs-cname                                                          The alternative names of the host the generated certificates
                                                                           should be for (current: [])
    --certs-node-fqdn                                                      The fqdn of the host the generated certificates
                                                                           should be for (current: "sat-6-15.example.com")
    --certs-server-ca-cert                                                 Path to the CA that issued the ssl certificates for https
                                                                           if not specified, the default CA will be used (current: UNDEF)
    --certs-server-cert                                                    Path to the ssl certificate for https
                                                                           if not specified, the default CA will generate one (current: UNDEF)
    --certs-server-cert-req                                                Path to the ssl certificate request for https
                                                                           if not specified, the default CA will generate one (current: UNDEF)
    --certs-server-key                                                     Path to the ssl key for https
                                                                           if not specified, the default CA will generate one (current: UNDEF)
    --certs-tar-file                                                       Use a tarball with certificates rather than generate
                                                                           new ones. This can be used on another node which is
                                                                           not the CA. (current: UNDEF)


= Module foreman:
    --foreman-db-manage                                                    If enabled, will install and configure the database server on this host (current: true)
    --foreman-email-delivery-method                                        Email delivery method (current: UNDEF)
    --foreman-email-reply-address                                          Email reply address for emails that Foreman is sending (current: UNDEF)
    --foreman-email-sendmail-arguments                                     The arguments to pass to the sendmail binary. Unused when SMTP delivery is used. (current: UNDEF)
    --foreman-email-sendmail-location                                      The location of the binary to call when sendmail is the delivery method. Unused when SMTP delivery is used. (current: UNDEF)
    --foreman-email-smtp-address                                           SMTP server hostname, when delivery method is SMTP (current: UNDEF)
    --foreman-email-smtp-authentication                                    SMTP authentication method (current: "none")
    --foreman-email-smtp-domain                                            SMTP HELO domain (current: UNDEF)
    --foreman-email-smtp-password                                          Password for SMTP server auth, if authentication is enabled (current: UNDEF)
    --foreman-email-smtp-port                                              SMTP port (current: 25)
    --foreman-email-smtp-user-name                                         Username for SMTP server auth, if authentication is enabled (current: UNDEF)
    --foreman-email-subject-prefix                                         Prefix to add to all outgoing email (current: UNDEF)
    --foreman-initial-admin-email                                          Initial E-mail address of the admin user (current: UNDEF)
    --foreman-initial-admin-first-name                                     Initial first name of the admin user (current: UNDEF)
    --foreman-initial-admin-last-name                                      Initial last name of the admin user (current: UNDEF)
    --foreman-initial-admin-locale                                         Initial locale (= language) of the admin user (current: UNDEF)
    --foreman-initial-admin-password                                       Initial password of the admin user, default is randomly generated (current: "efWE5TqTWYtPh9K8")
    --foreman-initial-admin-timezone                                       Initial timezone of the admin user (current: UNDEF)
    --foreman-initial-admin-username                                       Initial username for the admin user account, default is admin (current: "admin")
    --foreman-initial-location                                             Name of an initial location (current: "Default Location")
    --foreman-initial-organization                                         Name of an initial organization (current: "Default Organization")
    --foreman-ipa-authentication                                           Enable configuration for external authentication via IPA (current: false)
    --foreman-ipa-authentication-api                                       Enable configuration for external authentication via IPA for API (current: false)


= Module foreman_cli:
    --foreman-cli-foreman-url                                              URL on which Foreman runs (current: UNDEF)
    --foreman-cli-password                                                 Password for authentication (current: UNDEF)
    --foreman-cli-username                                                 Username for authentication (current: UNDEF)


= Module foreman_plugin_remote_execution_cockpit:
    --foreman-plugin-remote-execution-cockpit-origins                      Specify additional Cockpit Origins to configure cockpit.conf.
                                                                           The $foreman_url is included by default. (current: [])


= Module foreman_plugin_tasks:
    --foreman-plugin-tasks-automatic-cleanup                               Enable automatic task cleanup using a cron job (current: true)
    --foreman-plugin-tasks-backup                                          Enable creating a backup of cleaned up tasks in CSV format when automatic_cleanup is enabled (current: false)
    --foreman-plugin-tasks-cron-line                                       Cron line defining when the cleanup cron job should run (current: "45 19 * * *")


= Module foreman_proxy:
    --foreman-proxy-autosignfile                                           Hostname-Whitelisting only: Location of puppets autosign.conf (current: "/etc/puppetlabs/puppet/autosign.conf")
    --foreman-proxy-bind-host                                              Host to bind ports to, e.g. *, localhost, 0.0.0.0 (current: ["*"])
    --foreman-proxy-bmc                                                    Enable BMC feature (current: false)
    --foreman-proxy-bmc-default-provider                                   BMC default provider. (current: "ipmitool")
    --foreman-proxy-bmc-listen-on                                          BMC proxy to listen on https, http, or both (current: "https")
    --foreman-proxy-bmc-redfish-verify-ssl                                 BMC Redfish verify ssl. (current: true)
    --foreman-proxy-bmc-ssh-key                                            BMC SSH key location. (current: "/usr/share/foreman/.ssh/id_rsa")
    --foreman-proxy-bmc-ssh-powercycle                                     BMC SSH powercycle command. (current: "shutdown -r +1")
    --foreman-proxy-bmc-ssh-poweroff                                       BMC SSH poweroff command. (current: "shutdown +1")
    --foreman-proxy-bmc-ssh-poweron                                        BMC SSH poweron command. (current: "false")
    --foreman-proxy-bmc-ssh-powerstatus                                    BMC SSH powerstatus command. (current: "true")
    --foreman-proxy-bmc-ssh-user                                           BMC SSH user. (current: "root")
    --foreman-proxy-dhcp                                                   Enable DHCP feature (current: false)
    --foreman-proxy-dhcp-additional-interfaces                             Additional DHCP listen interfaces (in addition to dhcp_interface). Note: as opposed to dhcp_interface
                                                                           *no* subnet will be provisioned for any of the additional DHCP listen interfaces. Please configure any
                                                                           additional subnets using `dhcp::pool` and related resource types (provided by the theforeman/puppet-dhcp
                                                                           module). (current: [])
    --foreman-proxy-dhcp-config                                            DHCP config file path (current: "/etc/dhcp/dhcpd.conf")
    --foreman-proxy-dhcp-gateway                                           DHCP pool gateway (current: UNDEF)
    --foreman-proxy-dhcp-interface                                         DHCP listen interface (current: "eth0")
    --foreman-proxy-dhcp-ipxe-bootstrap                                    Enable or disable iPXE bootstrap(discovery) feature (current: false)
    --foreman-proxy-dhcp-ipxefilename                                      iPXE DHCP "filename" value, If not specified, it's determined dynamically.
                                                                           When the templates feature is enabled, the template_url is used. (current: UNDEF)
    --foreman-proxy-dhcp-key-name                                          DHCP key name (current: UNDEF)
    --foreman-proxy-dhcp-key-secret                                        DHCP password (current: UNDEF)
    --foreman-proxy-dhcp-leases                                            DHCP leases file (current: "/var/lib/dhcpd/dhcpd.leases")
    --foreman-proxy-dhcp-listen-on                                         DHCP proxy to listen on https, http, or both (current: "https")
    --foreman-proxy-dhcp-managed                                           The DHCP daemon is managed by this module (current: true)
    --foreman-proxy-dhcp-nameservers                                       DHCP nameservers, comma-separated (current: "default")
    --foreman-proxy-dhcp-netmask                                           DHCP server netmask value, defaults otherwise to value based on IP of dhcp_interface (current: UNDEF)
    --foreman-proxy-dhcp-network                                           DHCP server network value, defaults otherwise to value based on IP of dhcp_interface (current: UNDEF)
    --foreman-proxy-dhcp-node-type                                         DHCP node type (current: "standalone")
    --foreman-proxy-dhcp-omapi-port                                        DHCP server OMAPI port (current: 7911)
    --foreman-proxy-dhcp-option-domain                                     DHCP use the dhcpd config option domain-name (current: ["example.com"])
    --foreman-proxy-dhcp-peer-address                                      The other DHCP servers address (current: UNDEF)
    --foreman-proxy-dhcp-ping-free-ip                                      Perform ICMP and TCP ping when searching free IPs from the pool. This makes
                                                                           sure that active IP address is not suggested as free, however in locked down
                                                                           network environments this can cause no free IPs. (current: true)
    --foreman-proxy-dhcp-provider                                          DHCP provider for the DHCP module (current: "isc")
    --foreman-proxy-dhcp-pxefilename                                       DHCP "filename" value, defaults otherwise to pxelinux.0 (current: "pxelinux.0")
    --foreman-proxy-dhcp-pxeserver                                         DHCP "next-server" value, defaults otherwise to IP of dhcp_interface (current: UNDEF)
    --foreman-proxy-dhcp-range                                             Space-separated DHCP pool range (current: UNDEF)
    --foreman-proxy-dhcp-search-domains                                    DHCP search domains option (current: UNDEF)
    --foreman-proxy-dhcp-server                                            Address of DHCP server to manage (current: "127.0.0.1")
    --foreman-proxy-dhcp-subnets                                           Subnets list to restrict DHCP management to (current: [])
    --foreman-proxy-dns                                                    Enable DNS feature (current: false)
    --foreman-proxy-dns-forwarders                                         DNS forwarders (current: [])
    --foreman-proxy-dns-interface                                          DNS interface (current: "eth0")
    --foreman-proxy-dns-listen-on                                          DNS proxy to listen on https, http, or both (current: "https")
    --foreman-proxy-dns-managed                                            The DNS daemon is managed by this module. Only supported for the nsupdate and nsupdate_gss DNS providers. (current: true)
    --foreman-proxy-dns-provider                                           DNS provider (current: "nsupdate")
    --foreman-proxy-dns-reverse                                            DNS reverse zone name (current: UNDEF)
    --foreman-proxy-dns-server                                             Address of DNS server to manage (current: "127.0.0.1")
    --foreman-proxy-dns-tsig-keytab                                        Kerberos keytab for DNS updates using GSS-TSIG authentication (current: "/etc/foreman-proxy/dns.keytab")
    --foreman-proxy-dns-tsig-principal                                     Kerberos principal for DNS updates using GSS-TSIG authentication (current: "foremanproxy/sat-6-15.example.com@EXAMPLE.COM")
    --foreman-proxy-dns-ttl                                                DNS default TTL override (current: 86400)
    --foreman-proxy-dns-zone                                               DNS zone name (current: "example.com")
    --foreman-proxy-ensure-packages-version                                control extra packages version, it's passed to ensure parameter of package resource (current: "installed")
    --foreman-proxy-foreman-base-url                                       Base Foreman URL used for REST interaction (current: "https://sat-6-15.example.com")
    --foreman-proxy-foreman-ssl-ca                                         SSL CA used to verify connections when accessing the Foreman API.
                                                                           When not specified, the ssl_ca is used instead. (current: "/etc/foreman-proxy/foreman_ssl_ca.pem")
    --foreman-proxy-foreman-ssl-cert                                       SSL client certificate used when accessing the Foreman API
                                                                           When not specified, the ssl_cert is used instead. (current: "/etc/foreman-proxy/foreman_ssl_cert.pem")
    --foreman-proxy-foreman-ssl-key                                        Corresponding key to a foreman_ssl_cert certificate
                                                                           When not specified, the ssl_key is used instead. (current: "/etc/foreman-proxy/foreman_ssl_key.pem")
    --foreman-proxy-freeipa-config                                         Path to FreeIPA default.conf configuration file (current: "/etc/ipa/default.conf")
    --foreman-proxy-freeipa-remove-dns                                     Remove DNS entries from FreeIPA when deleting hosts from realm (current: true)
    --foreman-proxy-groups                                                 Array of additional groups for the foreman proxy user (current: [])
    --foreman-proxy-http                                                   Enable HTTP (current: false)
    --foreman-proxy-http-port                                              HTTP port to listen on (if http is enabled) (current: 8000)
    --foreman-proxy-httpboot                                               Enable HTTPBoot feature. In most deployments this requires HTTP to be enabled as well. (current: false)
    --foreman-proxy-keyfile                                                DNS server keyfile path (current: "/etc/rndc.key")
    --foreman-proxy-libvirt-connection                                     Connection string of libvirt DNS/DHCP provider (e.g. "qemu:///system") (current: "qemu:///system")
    --foreman-proxy-libvirt-network                                        Network for libvirt DNS/DHCP provider (current: "default")
    --foreman-proxy-log                                                    Foreman proxy log file, 'STDOUT', 'SYSLOG' or 'JOURNAL' (current: "/var/log/foreman-proxy/proxy.log")
    --foreman-proxy-log-buffer                                             Log buffer size (current: 2000)
    --foreman-proxy-log-buffer-errors                                      Additional log buffer size for errors (current: 1000)
    --foreman-proxy-log-level                                              Foreman proxy log level (current: "INFO")
    --foreman-proxy-logs                                                   Enable Logs (log buffer) feature (current: true)
    --foreman-proxy-logs-listen-on                                         Logs proxy to listen on https, http, or both (current: "https")
    --foreman-proxy-manage-puppet-group                                    Whether to ensure the $puppet_group exists.  Also ensures group owner of ssl keys and certs is $puppet_group
                                                                           Not applicable when ssl is false. (current: false)
    --foreman-proxy-oauth-consumer-key                                     OAuth key to be used for REST interaction (current: "6U7Z7AaqfFquPWHVGnqdhBwkwWM2x3Gs")
    --foreman-proxy-oauth-consumer-secret                                  OAuth secret to be used for REST interaction (current: "XmAPkNFEQGqL6QHDecYRdwifWfbhBZLj")
    --foreman-proxy-oauth-effective-user                                   User to be used for REST interaction (current: "admin")
    --foreman-proxy-puppet                                                 Enable Puppet module for environment imports and Puppet runs (current: false)
    --foreman-proxy-puppet-api-timeout                                     Timeout in seconds when accessing Puppet environment classes API (current: 30)
    --foreman-proxy-puppet-group                                           Groups of Foreman proxy user (current: "puppet")
    --foreman-proxy-puppet-listen-on                                       Protocols for the Puppet feature to listen on (current: "https")
    --foreman-proxy-puppet-ssl-ca                                          SSL CA used to verify connections when accessing the Puppet master API (current: "/etc/puppetlabs/puppet/ssl/certs/ca.pem")
    --foreman-proxy-puppet-ssl-cert                                        SSL certificate used when accessing the Puppet master API (current: "/etc/puppetlabs/puppet/ssl/certs/sat-6-15.example.com.pem")
    --foreman-proxy-puppet-ssl-key                                         SSL private key used when accessing the Puppet master API (current: "/etc/puppetlabs/puppet/ssl/private_keys/sat-6-15.example.com.pem")
    --foreman-proxy-puppet-url                                             URL of the Puppet master itself for API requests (current: "https://sat-6-15.example.com:8140")
    --foreman-proxy-puppetca                                               Enable Puppet CA feature (current: false)
    --foreman-proxy-puppetca-listen-on                                     Protocols for the Puppet CA feature to listen on (current: "https")
    --foreman-proxy-puppetca-tokens-file                                   Token-Whitelisting only: Location of the tokens.yaml (current: "/var/lib/foreman-proxy/tokens.yml")
    --foreman-proxy-puppetdir                                              Puppet var directory (current: "/etc/puppetlabs/puppet")
    --foreman-proxy-realm                                                  Enable realm management feature (current: false)
    --foreman-proxy-realm-keytab                                           Kerberos keytab path to authenticate realm updates (current: "/etc/foreman-proxy/freeipa.keytab")
    --foreman-proxy-realm-listen-on                                        Realm proxy to listen on https, http, or both (current: "https")
    --foreman-proxy-realm-principal                                        Kerberos principal for realm updates (current: "realm-proxy@EXAMPLE.COM")
    --foreman-proxy-realm-provider                                         Realm management provider (current: "freeipa")
    --foreman-proxy-register-in-foreman                                    Register proxy back in Foreman (current: true)
    --foreman-proxy-registered-name                                        Proxy name which is registered in Foreman (current: "sat-6-15.example.com")
    --foreman-proxy-registered-proxy-url                                   Proxy URL which is registered in Foreman (current: UNDEF)
    --foreman-proxy-registration                                           Enable Registration feature (current: false)
    --foreman-proxy-registration-listen-on                                 Registration proxy to listen on https, http, or both (current: "https")
    --foreman-proxy-ssl                                                    Enable SSL, ensure feature is added with "https://" protocol if true (current: true)
    --foreman-proxy-ssl-ca                                                 SSL CA to validate the client certificates used to access the proxy (current: "/etc/foreman-proxy/ssl_ca.pem")
    --foreman-proxy-ssl-cert                                               SSL certificate to be used to run the foreman proxy via https. (current: "/etc/foreman-proxy/ssl_cert.pem")
    --foreman-proxy-ssl-disabled-ciphers                                   List of OpenSSL cipher suite names that will be disabled from the default (current: [])
    --foreman-proxy-ssl-key                                                Corresponding key to a ssl_cert certificate (current: "/etc/foreman-proxy/ssl_key.pem")
    --foreman-proxy-ssl-port                                               HTTPS port to listen on (if ssl is enabled) (current: 9090)
    --foreman-proxy-ssldir                                                 Puppet CA SSL directory (current: "/etc/puppetlabs/puppet/ssl")
    --foreman-proxy-template-url                                           URL a client should use for provisioning templates (current: "http://sat-6-15.example.com:8000")
    --foreman-proxy-templates                                              Enable templates feature (current: false)
    --foreman-proxy-templates-listen-on                                    Templates proxy to listen on https, http, or both (current: "both")
    --foreman-proxy-tftp                                                   Enable TFTP feature (current: false)
    --foreman-proxy-tftp-dirs                                              Directories to be create in $tftp_root (current: UNDEF)
    --foreman-proxy-tftp-listen-on                                         TFTP proxy to listen on https, http, or both (current: "https")
    --foreman-proxy-tftp-manage-wget                                       If enabled will install the wget package (current: true)
    --foreman-proxy-tftp-managed                                           The TFTP daemon is managed by this module. (current: true)
    --foreman-proxy-tftp-replace-grub2-cfg                                 Determines if grub2.cfg will be replaced (current: false)
    --foreman-proxy-tftp-root                                              TFTP root directory (current: "/var/lib/tftpboot")
    --foreman-proxy-tftp-servername                                        Defines the TFTP Servername to use, overrides the name in the subnet declaration (current: UNDEF)
    --foreman-proxy-tls-disabled-versions                                  List of TLS versions that will be disabled from the default (current: [])
    --foreman-proxy-trusted-hosts                                          Only hosts listed will be permitted, empty array to disable authorization (current: ["sat-6-15.example.com"])
    --foreman-proxy-version                                                foreman package version, it's passed to ensure parameter of package resource
                                                                           can be set to specific version number, 'latest', 'present' etc. (current: "present")


= Module foreman_proxy_content:
    --foreman-proxy-content-enable-ansible                                 Enable the Ansible content feature. This allows syncing, managing, and serving Ansible content. (current: true)
    --foreman-proxy-content-enable-deb                                     Enable the Debian content feature. This allows syncing, managing, and serving Debian content. (current: false)
    --foreman-proxy-content-enable-docker                                  Enable the container content feature. This allows syncing, managing, and serving container content. (current: true)
    --foreman-proxy-content-enable-file                                    Enable the file content feature. This allows syncing, managing, and serving file content. (current: true)
    --foreman-proxy-content-enable-ostree                                  Enable the OSTree content feature. This allows syncing, managing, and serving OSTree content. (current: false)
    --foreman-proxy-content-enable-python                                  Enable the Python content feature. This allows syncing, managing, and serving Python content. (current: false)
    --foreman-proxy-content-enable-yum                                     Enable the RPM content feature. This allows syncing, managing and serving RPM content to be consumed by package managers like yum and dnf. (current: true)
    --foreman-proxy-content-pulpcore-mirror                                Deploy Pulp to be used as a mirror (current: false)


= Module foreman_proxy_plugin_dhcp_infoblox:
    --foreman-proxy-plugin-dhcp-infoblox-dns-view                          The DNS view to use (current: "default")
    --foreman-proxy-plugin-dhcp-infoblox-network-view                      The network view to use (current: "default")
    --foreman-proxy-plugin-dhcp-infoblox-password                          The password of the Infoblox user (current: UNDEF)
    --foreman-proxy-plugin-dhcp-infoblox-record-type                       Record type to manage (current: "fixedaddress")
    --foreman-proxy-plugin-dhcp-infoblox-used-ips-search-type              The search type for used ips (current: "record_type")
    --foreman-proxy-plugin-dhcp-infoblox-username                          The username of the Infoblox user (current: UNDEF)


= Module foreman_proxy_plugin_dhcp_remote_isc:
    --foreman-proxy-plugin-dhcp-remote-isc-dhcp-config                     DHCP config file path (current: "/etc/dhcp/dhcpd.conf")
    --foreman-proxy-plugin-dhcp-remote-isc-dhcp-leases                     DHCP leases file (current: "/var/lib/dhcpd/dhcpd.leases")
    --foreman-proxy-plugin-dhcp-remote-isc-key-name                        DHCP key name (current: UNDEF)
    --foreman-proxy-plugin-dhcp-remote-isc-key-secret                      DHCP password (current: UNDEF)
    --foreman-proxy-plugin-dhcp-remote-isc-omapi-port                      DHCP server OMAPI port (current: 7911)


= Module foreman_proxy_plugin_discovery:
    --foreman-proxy-plugin-discovery-install-images                        Download and extract the discovery image (current: false)


= Module foreman_proxy_plugin_dns_infoblox:
    --foreman-proxy-plugin-dns-infoblox-dns-server                         The address of the Infoblox server (current: UNDEF)
    --foreman-proxy-plugin-dns-infoblox-dns-view                           The Infoblox DNS View (current: "default")
    --foreman-proxy-plugin-dns-infoblox-password                           The password of the Infoblox user (current: UNDEF)
    --foreman-proxy-plugin-dns-infoblox-username                           The username of the Infoblox user (current: UNDEF)


= Module foreman_proxy_plugin_openscap:
    --foreman-proxy-plugin-openscap-ansible-module                         Ensure the Ansible module (current: false)
    --foreman-proxy-plugin-openscap-contentdir                             Directory where OpenSCAP content XML are stored
                                                                           So we will not request the XML from Foreman each time (current: "/var/lib/foreman-proxy/openscap/content")
    --foreman-proxy-plugin-openscap-corrupted-dir                          Directory where corrupted OpenSCAP report XML are stored (current: "/var/lib/foreman-proxy/openscap/corrupted")
    --foreman-proxy-plugin-openscap-failed-dir                             Directory where OpenSCAP report XML are stored
                                                                           In case sending to Foreman succeeded, yet failed to save to reportsdir (current: "/var/lib/foreman-proxy/openscap/failed")
    --foreman-proxy-plugin-openscap-openscap-send-log-file                 Log file for the forwarding script (current: "/var/log/foreman-proxy/openscap-send.log")
    --foreman-proxy-plugin-openscap-proxy-name                             Proxy name to send to Foreman with parsed report
                                                                           Foreman matches it against names of registered proxies to find the report source (current: UNDEF)
    --foreman-proxy-plugin-openscap-puppet-module                          Ensure the Puppet module. This only makes sense if Puppetserver runs on the same machine. (current: false)
    --foreman-proxy-plugin-openscap-reportsdir                             Directory where OpenSCAP report XML are stored
                                                                           So Foreman can request arf xml reports (current: "/var/lib/foreman-proxy/openscap/reports")
    --foreman-proxy-plugin-openscap-spooldir                               Directory where OpenSCAP audits are stored
                                                                           before they are forwarded to Foreman (current: "/var/spool/foreman-proxy/openscap")
    --foreman-proxy-plugin-openscap-timeout                                Timeout for sending ARF reports to foreman (current: 60)


= Module foreman_proxy_plugin_remote_execution_script:
    --foreman-proxy-plugin-remote-execution-script-cockpit-integration     Enables/disables Cockpit integration (current: true)
    --foreman-proxy-plugin-remote-execution-script-mode                    Operation Mode of the plugin. (current: "ssh")


= Module foreman_proxy_plugin_shellhooks:
    --foreman-proxy-plugin-shellhooks-directory                            Absolute path to directory with executables (current: "/var/lib/foreman-proxy/shellhooks")


= Module puppet:
    --puppet-ca-server                                                     Use a different ca server. Should be either
                                                                           a string with the location of the ca_server
                                                                           or 'false'. (current: UNDEF)
    --puppet-show-diff                                                     Show and report changed files with diff output (current: false)

Only commonly used options have been displayed.
Use --full-help to view the complete list.
```

Reply:
You're right. Reminding users of --help is still useful at this point (in case they have questions about some of the CLI options used in the example command), so let's try rephrasing the reference.